macros/utils/get_sde_or_context.sql

```diff
@@ -1,13 +1,15 @@
-{% macro get_sde_or_context(schema, identifier, lower_limit, upper_limit, prefix = none) %}
-    {{ return(adapter.dispatch('get_sde_or_context', 'snowplow_utils')(schema, identifier, lower_limit, upper_limit, prefix)) }}
+{% macro get_sde_or_context(schema, identifier, lower_limit, upper_limit, prefix = none, single_entity = true) %}
+    {{ return(adapter.dispatch('get_sde_or_context', 'snowplow_utils')(schema, identifier, lower_limit, upper_limit, prefix, single_entity)) }}
 {% endmacro %}
 
-{% macro default__get_sde_or_context() %}
-    {% do exceptions.raise_compiler_error('Macro get_sde_or_context is only for Postgres or Redshift, it is not supported for' ~ target.type) %}
+{% macro default__get_sde_or_context(schema, identifier, lower_limit, upper_limit, prefix = none, single_entity = true) %}
```

Comment:
```suggestion
{% macro default__get_sde_or_context(schema, identifier, lower_limit, upper_limit, prefix, single_entity) %}
```

Not sure if those default args are needed here, given they're already passed in the dispatch

Reply:
Technically it is possible that someone calls the specific macro by name rather than the generic version, so it is best practise to keep them in here as well.